modules/peers.js

```diff
@@ -267,11 +267,11 @@ __private.dbLoad = function (cb) {
 			}
 		}, function (err) {
 			library.logger.trace('Peers->dbLoad Peers discovered', {updated: updated, total: rows.length});
-			return setImmediate(cb);
+			return setImmediate(cb, err);
 		});
 	}).catch(function (err) {
 		library.logger.error('Import peers from database failed', {error: err.message || err});
-		return setImmediate(cb);
+		return setImmediate(cb, err);
```

Comment:
That is not change of behavior because `err` is discarded at to level function (`async.series`).

Reply:
So I will leave as is ;)